powerdns/models/powerdns.py

```diff
@@ -390,6 +390,53 @@ class Meta:
         verbose_name = _("record")
         verbose_name_plural = _("records")
 
+    def request_factory(operation):
```

Comment:
This copy-paste from: https://github.com/allegro/django-powerdns-dnssec/blob/develop/powerdns/models/powerdns.py#L126
With only one small change:
```
 +                ) and
 +                can_edit(get_current_user(), self) or
 +                can_delete(get_current_user(), self)
```
Why gonna get rid of it all when admin become simple CRUD.
No point of changing it, IMO :)